### ISSUE TITLE
Towny blocks message 

### ISSUE THREAD

**Issue by jordanwilliams1** [ElgarL/Towny#179]
Can you make an option to disable this? We sell towny blocks in our Point shop and we like having chat all smooth when people buy stuff... Unfonrently towny loves to give a massive blue message.

https://i.gyazo.com/cd6bc60091b386655ea56221c68e5b57.png

Infact, I've seen other towny servers sell towny blocks still... They have just ediited the message out of the plugin. I Just don't see the point in this.

**Comment by maxlehot1234** [comment]
@jordanwilliams1 Yes you can use this: https://github.com/GraxCode/JByteMod/releases

Since the actual dev of towny will NEVER remove this message, you can change it for something you want using the JByteMod software ;)

Have fun and hope you will receive an email for the link, since it will removed in few second by ElgarL I think :P
